server/util/clickhouse/schema/schema.go

```diff
@@ -215,6 +215,9 @@ type Execution struct {
 	RequestedIsolationType string
 	EffectiveIsolationType string `gorm:"type:LowCardinality(String)"` // This values comes from the executor
 
+	RequestedTimeoutMsec int64
+	EffectiveTimeoutMsec int64
```

Comment:
I think it would be fine to use `usec` for consistency with the other fields - people are mostly using second-level precision for these so I'd expect the trailing digits will mostly be 0 and these should compress well

Reply:
yeah, I was debating which way we go. I'm not a fan of using microseconds for multiple reasons, but I guess it's better to be consistent.